jumpscale/tools/nginx/nginxserver.py

```diff
@@ -0,0 +1,30 @@
+from jumpscale.god import j
+from jumpscale.core.base import Base, fields
+
+
+class NginxServer(Base):
+    name = fields.String(default="main")
+    path = fields.String(default="~/sandbox/cfg/nginx/main/nginx.conf")
+
+    def install(self):
+        j.sals.process.execute("apt install nginx", showout=True, die=False)
+
+    def start(self):
+        nginx = j.sals.nginx.get(self.name)
+        nginx.configure()
+        nginx.save()
+        cmd = j.tools.startupcmd.get(self.name)
+        cmd.start_cmd = f"nginx -c {self.path}"
```

Comment:
You are starting nginx seperate from systemd, and then you are using to stop this won't work

Reply:
self.path is changed to self.config_path